tests_v2/test_terminal_focus_routing.py

```
@@ -114,6 +114,7 @@ def _assert_routed_to_surface(c: cmux, expected_surface_id: str, panel_id: str)
 
 
 def main() -> int:
+    focus_file = Path(tempfile.mktemp(prefix="cmux_focus_routing_", suffix=".txt"))
```

Ruff (0.15.17)

[error] Use of insecure and deprecated function (`mktemp`)

(S306)